templates/log-in.html

```diff
@@ -1,5 +1,77 @@
+<!DOCTYPE html>
 <html>
 
+<head>
+    <title>Log In</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Bootstrap 4 CDN link -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <!-- font-awesome 5 CDN link -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css"
+        integrity="sha512-YWzhKL2whUzgiheMoBFwW8CKV4qpHQAEuvilg9FAn5VJUDwKZZxkJNuGM4XkWuk94WCrrwslk8yWNGmY1EduTA=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="/src/style.css">
+</head>
+
+<body>
+    <div class="container">
+        <div class="myCard">
+            <div class="row">
+                <div class="col-md-6">
+                    <div class="myLeftCtn">
+                        <form action="POST" class="myForm text-center">
+                            <header>Log In to Habit Tracker</header>
+                            <div class="form-goup">
+                                <i class="fas fa-user"></i>
+                                <input class="myInput" type="text" placeholder="Username" id="username" required>
+                            </div>
+                            <br>
+
+                            <div class="form-group">
+                                <i class="fas fa-lock"></i>
+                                <input class="myInput" type="password" placeholder="Password" id="password" required>
+                            </div>
+
+                            <input type="submit" class="butt" value="LOG IN"><BR></BR>
+                            <p>OR</p>
+                            <input type="submit" class="butt" value="CREATE AN ACCOUNT">
```

Comment:
This should be the same format as:
<p>Already have an account? <a href="{{ url_for('login') }}">Log In</a></p>
and stay outside of the form.